x-pack/legacy/plugins/apm/public/components/shared/Summary/UserAgentSummaryItem.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { UserAgent } from '../../../../typings/es_schemas/raw/fields/UserAgent';
+
+type UserAgentSummaryItemProps = UserAgent;
+
+const Version = styled('span')`
+  color: ${theme.textColors.subdued};
+  font-size: ${theme.euiFontSizeS};
+`;
+
+export function UserAgentSummaryItem({
+  name,
+  version
+}: UserAgentSummaryItemProps) {
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetails.userAgentLabel', {
+        defaultMessage: 'User agent'
```

Comment:
Perhaps just "User agent & version" as the default message as that's what is expected to display.